docs/config.md

```diff
@@ -200,6 +200,7 @@ due to lack of stable semantic convention.
 | ID        | Instrumented library                                                                                                            | Supported versions | Instrumentation type   | Status                                                                                                                            |
 |-----------|---------------------------------------------------------------------------------------------------------------------------------|--------------------|------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
 | `ILOGGER` | [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) **Not supported on .NET Framework** | ≥8.0.0             | bytecode or source [1] | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `LOG4NET` | [log4net](https://www.nuget.org/packages/log4net)                                                                               | ≥2.0.0 && < 4.0.0  | bytecode               | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
```

Comment:
I think that the mapping (translation form log4net to OTel Logs) should be documented.

Reply:
Added in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3825/commits/735deacf9608861c6d797b80cb1851f45d9d1be3